Configuration/TCA/tx_pxaproductmanager_domain_model_product.php

```diff
@@ -374,6 +374,11 @@
                             'disabled' => true,
                         ],
                     ],
+                    'suggestOptions' => [
+                        'tx_pxaproductmanager_domain_model_product' => [
+                            'searchCondition' => 'sys_language_uid in (0,-1) AND parent = 0',
```

Comment:
* Won't `parent = 0` mean it only allows you to select parents who do not themselves have parents?
* There should probably be a check for inheritance loops here? Granddad being your own child kind of thing.

Reply:
the plan with `parent = 0` was that it shouldn't be possible to select a product that already have a parent selected.

Reply:
Shouln't that be possible, though? Having $product->getParent()->getParent()->getParent() ad infinitum

Reply:
I'll call you.